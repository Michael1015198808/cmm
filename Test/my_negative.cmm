int ___() {
    -12;
    - 12;
}
