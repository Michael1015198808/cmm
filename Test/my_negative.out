Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: ___
        LP
        RP
      CompSt (1)
        LC
        StmtList (2)
          Stmt (2)
            Exp (2)
              MINUS
              Exp (2)
                INT: 12
            SEMI
          StmtList (3)
            Stmt (3)
              Exp (3)
                MINUS
                Exp (3)
                  INT: 12
              SEMI
        RC
